pseudonymsys/org_issue.go

```diff
@@ -67,8 +82,6 @@ func (org *OrgCredentialIssuer) VerifyAuthentication(z *big.Int) (
 
 		return x11, x12, x21, x22, A, B, nil
 	} else {
-		// TODO: close the session
-
 		err := errors.New("Authentication with organization failed")
 		return nil, nil, nil, nil, nil, nil, err
```

Comment:
We might define common.DoubleBigInt and return three DoubleBigInts, but I am not sure this is much better :smiley: